x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/dataframe/process/results/ModelMetadata.java

```diff
@@ -24,23 +25,29 @@
 
     public static final ParseField TOTAL_FEATURE_IMPORTANCE = new ParseField("total_feature_importance");
     public static final ParseField FEATURE_IMPORTANCE_BASELINE = new ParseField("feature_importance_baseline");
+    public static final ParseField HYPERPARAMETERS = new ParseField("hyperparameters");
 
     @SuppressWarnings("unchecked")
     public static final ConstructingObjectParser<ModelMetadata, Void> PARSER = new ConstructingObjectParser<>(
         "trained_model_metadata",
-        a -> new ModelMetadata((List<TotalFeatureImportance>) a[0], (FeatureImportanceBaseline) a[1]));
+        a -> new ModelMetadata((List<TotalFeatureImportance>) a[0], (FeatureImportanceBaseline) a[1], (List<Hyperparameters>) a[2]));
 
     static {
         PARSER.declareObjectArray(constructorArg(), TotalFeatureImportance.STRICT_PARSER, TOTAL_FEATURE_IMPORTANCE);
         PARSER.declareObject(optionalConstructorArg(), FeatureImportanceBaseline.STRICT_PARSER, FEATURE_IMPORTANCE_BASELINE);
+        PARSER.declareObjectArray(constructorArg(), Hyperparameters.STRICT_PARSER, HYPERPARAMETERS);
+
     }
 
     private final List<TotalFeatureImportance> featureImportances;
     private final FeatureImportanceBaseline featureImportanceBaseline;
+    private final List<Hyperparameters> hyperparameters;
 
-    public ModelMetadata(List<TotalFeatureImportance> featureImportances, FeatureImportanceBaseline featureImportanceBaseline) {
+    public ModelMetadata(List<TotalFeatureImportance> featureImportances, FeatureImportanceBaseline featureImportanceBaseline, 
+    List<Hyperparameters> hyperparameters) {
```

Comment:
```suggestion
    public ModelMetadata(List<TotalFeatureImportance> featureImportances, 
        FeatureImportanceBaseline featureImportanceBaseline, 
        List<Hyperparameters> hyperparameters) {
```

Reply:
I don't see the change here. Is this a formatting issue?